test/extended/cluster/cl.go

```diff
@@ -129,25 +129,24 @@ var _ = g.Describe("[Feature:Performance][Serial][Slow] Load cluster", func() {
 
 		// Wait for builds and deployments to complete
 		for _, ns := range namespaces {
-			buildList, err := oc.BuildClient().Build().Builds(ns).List(metav1.ListOptions{})
+			buildList, err := oc.Client().Builds(ns).List(metav1.ListOptions{})
 			if err != nil {
 				e2e.Logf("Error listing builds: %v", err)
 			}
 			if len(buildList.Items) > 0 {
 				buildName := buildList.Items[0].Name
 				e2e.Logf("Waiting for build: %q", buildName)
-				err = exutil.WaitForABuild(oc.BuildClient().Build().Builds(ns), buildName, nil, nil, nil)
+				err = exutil.WaitForABuild(oc.Client().Builds(ns), buildName, nil, nil, nil)
 				if err != nil {
 					exutil.DumpBuildLogs(buildName, oc)
 				}
 				o.Expect(err).NotTo(o.HaveOccurred())
 				e2e.Logf("Build %q completed", buildName)
 
 				// deploymentName is buildName without the -1 suffix
-				deploymentName := buildName[:len(buildName)-2]
+				deploymentName := buildName[:len(buildName) - 2]
 				e2e.Logf("Waiting for deployment: %q", deploymentName)
-				err = exutil.WaitForDeploymentConfig(oc.KubeClient(), oc.AppsClient().Apps(), ns, deploymentName, 1, oc)
-				o.Expect(err).NotTo(o.HaveOccurred())
+				oc.Run("rollout status deployment/" + deploymentName)
```

Comment:
I used "oc.Run(rollout..." in order to wait the deployment to be ready.
I see that oc.Run dose not return an error, which is might be tricky, but I assume, since oc rollout retuen the stdout, I guess the functoion will return when it's ready.

also, I belive we can use the entire scope to use rollout from line 130.
@stevekuznetsov there rollout combination that cover everything per namespace e.g builds, buildconfigs, deployment and so on
instead of using rollout per kind.